todo_app/tests/test_viewModel.py

```diff
@@ -7,7 +8,7 @@ def test_CheckTodoList():
     items = []
     items.append(Item(1,ItemStatus.TODO))
 
-    view = ViewModel(items)
+    view = ViewModel(items,User())
```

Comment:
This is definitely fine, but could be a sign that you might want to set an empty user as a default argument for the ViewModel